lms/djangoapps/certificates/views/webview.py

```diff
@@ -540,6 +542,10 @@ def render_html_view(request, user_id, course_id):
     # Append microsite overrides
     _update_microsite_context(context, configuration)
 
+    # Update context with brand specific urls
```

Comment:
Reword to s 'Add certificate header/footer data to current context' -- at this point we do not know what the functions will provide.